e2e-tests/test/evm-apis.test.ts

```diff
@@ -56,13 +56,13 @@ describe("evm_setNextBlockTimestamp", function () {
     const userWallet = Wallet.createRandom().connect(provider);
 
     // Act
-    await provider.send("evm_setNextBlockTimestamp", [expectedTimestamp]);
+    await provider.send("evm_setNextBlockTimestamp", [expectedTimestamp.toString(16)]);
 
     await wallet.sendTransaction({
       to: userWallet.address,
       value: ethers.utils.parseEther("0.1"),
     });
-    expectedTimestamp += 2; // New transaction will add two blocks
+    expectedTimestamp += 1; // New transaction will add two blocks
 
     // Assert
     const newBlockTimestamp = (await provider.getBlock("latest")).timestamp;
```

Comment:
a bit inconsistent to use `config_getCurrentTimestamp ` above and `provider.getBlock("latest")` here.

Reply:
I believe test code shouldn't be consistent - it tests more when it varies...